frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.module.css

```diff
@@ -42,9 +42,24 @@
 .shoutoutFrom {
   color: gray;
   display: inline-block;
+  margin-top: 0 !important;
   align-self: left;
 }
 
+.shoutoutHide {
+  display: flex;
+  justify-content: space-between !important;
+  margin-top: 0 !important;
+  margin-bottom: 0 !important;
+}
+
+.shoutoutMessage {
+  color: black;
+  font-weight: bold;
+  margin-top: 0 !important;
+  font-size: 15px !important;
```

Comment:
I think it's best to avoid `!important` if possible

Reply:
I just took out a few, but the rest directly impact the formatting of the page.